### PR TITLE
feature/admin_login

### PR DESCRIPTION
**Pull request by Rodrigo-Miranda-0**
#### Board:
* [Ticket #AvvcA2XH](https://trello.com/c/AvvcA2XH/30-3-as-an-admin-i-should-be-able-to-login-in-the-admin-page)
---
#### Description:

* Installs the ActiveAdmin gem
---
#### Notes:
*
---
#### Tasks:
- [x] Install ActiveAdmin gem
---
#### Risk:
*
---
#### Preview:

### CODE REVIEW

**Review by jpascual1994 — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -42,6 +42,12 @@ gem 'active_storage_base64', '~> 2.0.0'
 
 gem 'activerecord-postgis-adapter', '~> 8.0.1'
 
+gem 'activeadmin'
+
+gem 'sprockets-rails'
+
+gem 'sassc-rails'
```
Comment (juanmferreira93) [inline Gemfile]: please add gem versions

File: app/admin/dashboard.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    #       end
+    #     end
+    #   end
+
+    #   column do
+    #     panel "Info" do
+    #       para "Welcome to ActiveAdmin."
+    #     end
+    #   end
+    # end
```
Comment (juanmferreira93) [inline app/admin/dashboard.rb]: remove unnecessary comments

File: spec/models/admin_user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+#
+# Indexes
+#
+#  index_admin_users_on_email                 (email) UNIQUE
+#  index_admin_users_on_reset_password_token  (reset_password_token) UNIQUE
+#
+require 'rails_helper'
+
+RSpec.describe AdminUser, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (juanmferreira93) [inline spec/models/admin_user_spec.rb]: I suggest to not test Admin features as it's a gem.. so you could remove that file

**Review by juanmferreira93 — APPROVED** [review]
Leave the approve but please address the comments

**Pull request merged**